extensions/pkg/controller/healthcheck/worker/nodes.go

```diff
@@ -180,7 +180,7 @@ func (h *DefaultHealthChecker) Check(ctx context.Context, request types.Namespac
 			return &healthcheck.SingleCheckResult{
 				Status:               status,
 				Detail:               err.Error(),
-				Codes:                gardencorev1beta1helper.DeprecatedDetermineErrorCodes(err),
+				Codes:                []gardencorev1beta1.ErrorCode{}, // User error will not occure here.
```

Comment:
Do we still need to mention `Codes` in the struct if it's empty? or we can drop this line?

Reply:
If you need, then
```suggestion
			        Codes:                []gardencorev1beta1.ErrorCode{}, // User error will not occur here.
```